src/components/Fretboard/Fretboard.module.css

```
@@ -40,21 +40,24 @@
 
 /* Tablet and desktop: enable native scroll on the fretboard (same as mobile)
    so an overflowing board scrolls horizontally instead of being clipped by the
-   base `overflow: clip` rule. */
-:global(.app-container[data-layout-tier="tablet"]) .fretboard-wrapper,
-:global(.app-container[data-layout-tier="desktop"]) .fretboard-wrapper {
+   base `overflow: clip` rule.
+   Scoped by the data-layout-tier attribute (not .app-container) so the rules
+   apply under both shells: MainLayoutWrapper (.app-container) and MobileShell
+   both set data-layout-tier on an ancestor. */
+:global([data-layout-tier="tablet"]) .fretboard-wrapper,
```

Stylelint (17.12.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)